Makefile

```diff
@@ -76,14 +77,14 @@ docker-push:
 	docker push ${IMG}
 
 #password can be found at https://connect.redhat.com/project/4805411/view
-redhat-image-scan: docker-build docker-push
+redhat-image-scan: #docker-build docker-push
```

Comment:
@cliveseldon is the comment here deliberate? Fine if you want to do it that way, just not sure if it's deliberate

Reply:
no its a typo from testing. will fix.